product_docs/docs/biganimal/release/administering_cluster/aws_secrets_manager_integration.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: "AWS Secrets Manager integration"
+---
+
+With BigAnimal, you can use AWS Secrets Manager. AWS Secrets Manager helps you manage, retrieve, and rotate database credentials, Access keys, and other Secrets throughout their lifecycle.
+
+To create a secret manager:
+
+1. Create a PostgreSQL cluster on BigAnimal portal.
+
+1. Create and save an [Access key](../reference/access_key/#create-your-personal-access-key).
+
+1. Create secret in application Secrets Manager for your **psql** credentials. 
+
+   Create the secret manager using a Lambda script or using the AWS console:
+
+   - Lambda script:
+
+     ```shell
+     import boto3
+     import json
+
+     def create_secret(secret_name, username, password, database, host):
+                client = boto3.client('secretsmanager')
+
+                secret_string = json.dumps({
+           "username": username,
+           "password": password,
+           "engine": "mysql",
+           "host": host,
+           "dbname": database,
+           "port": 3306
+         })
+
+    response = client.create_secret(
+        Name=secret_name,
+        SecretString=secret_string
+    )
+
+    return response
+    ```
+    
+    Using the created secret:
+
+    ```shell
+    create_secret('mySecretName', 'myUsername', 'myPassword', 'myDatabase', 'myHost')
+    ```
+
+   - AWS console:
+   
+     1. Search for Secret Manager under services.
+     1. Select **Store a new secret**.
+     1. Select **Credentials for other databases** on **Choose secret type** page and provide:
+        - Username
+        - Password
+        - Encryption key
+        - Database
+          Provide the server address, database name and port as per the selected database engine. Select **Next**.
```

Comment:
Should this be a step? Or is this part of the Database bullet? If the latter, then Select Next should be on its own line or its own step. Couldn't make out what was intended here.

Reply:
it's part of the database bullet, so Select Next should be its own step